recipes/libxsmm/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+make -j${CPU_COUNT} PREFIX=${PREFIX}
```

Comment:
I think PREFIX doesn't need to be set explicitly:
```suggestion
make -j${CPU_COUNT}
```